### PR TITLE
Update and copy-edit docs. Add CONTRIBUTING file

### PR DESCRIPTION
**Pull request by Elchi3**

### CODE REVIEW

File: compat-data-schema.md [hunk 1]
```diff
@@ -1,334 +1,306 @@
...
+* `nodejs` Node.js JavaScript runtime built on Chrome's V8 JavaScript engine,
+* `opera`, the Opera browser (desktop), based on Blink since Opera 15,
+* `opera_android`, the Opera browser (Android version),
+* `safari`, Apple Safari, on Mac OS,
+* `safari_ios`, Apple Safari, on iOS.
+
+No browser identifier is mandatory.
+
+#### The `support_statement` object
+This object is the key element for each browser.
```
Comment (wbamberg) [inline compat-data-schema.md]: This is a strange sentence. "key element" makes it sound like it has a technical meaning, like a database key. We could say instead "The `support_statement` object describes the support provided by a single browser type for the given subfeature ".

File: compat-data-schema.md [hunk 2]
```diff
@@ -1,334 +1,306 @@
...
+It is an array of `simple_support_statement` objects, but if there
+is only one of them, the array can be omitted.
+
+
+Example of an `array_support_statement` compat object (with 2 entries):
```
Comment (wbamberg) [inline compat-data-schema.md]: Technically this is an example of a`support` object containing support information for a single browser, that's expressed using an `array_support_statement` compat object containing 2 entries. IDK if that matters.

File: compat-data-schema.md [hunk 3]
```diff
@@ -1,334 +1,306 @@
...
+```json
+"support": {
+  "firefox": [
+    {
+      "version_added": "6.0"
+    },
+    {
+      "prefix": "-moz-",
+      "version_added": "3.5",
+      "version_removed": "9.0"
```
Comment (wbamberg) [inline compat-data-schema.md]: if we are deprecating "x.0" in favour of "x", this should be updated.

File: compat-data-schema.md [hunk 4]
```diff
@@ -1,334 +1,306 @@
...
+}
+```
+
+### Compat data in `simple_support_statement` objects
+This is the core object containing the compatibility information for a browser.
+It consist of the following properties:
+
+#### `version_added`
+This is the only mandatory property and it contains a string with the version
+number indicating when a sub-feature ha been added (and is therefore supported).
```
Comment (wbamberg) [inline compat-data-schema.md]: ha->has

File: compat-data-schema.md [hunk 5]
```diff
@@ -1,334 +1,306 @@
...
+{
+  "version_added": false
+}
+```
+* Support unknown (default value, if browser omitted):
+```json
+{
+  "version_added" : null
+}
+```
```
Comment (wbamberg) [inline compat-data-schema.md]: Examples are amazingly helpful! I wonder if we should have examples in the Sub-features section too, that gets a bit abstract.

File: compat-data-schema.md [hunk 6]
```diff
@@ -1,334 +1,306 @@
...
+
+#### `version_removed`
+Contains a string with the version number the sub-feature is
+removed in. It may also be a Boolean value of (`true` or `false`), or the
+`null` value. If `version_added` is set to a Boolean or a `string`, `version_removed`
+default value is `false`; if it is `null`, the default value of `version_removed` is `null` too. Examples:
```
Comment (wbamberg) [inline compat-data-schema.md]: "if it is" -> "it" here is ambiguous and this sentence is hard to parse. Maybe:

* If `version_added` is set to `true`, `false`, or a string, `version_removed` defaults to `false`
* if`version_added` is set to `null`, the default value of `version_removed` is also `null`.

File: compat-data-schema.md [hunk 7]
```diff
@@ -1,334 +1,306 @@
...
-      {
-        "prefix": "-moz-",
-        "version_added": "3.5"
-      }
-
-#### `"alternative_name"`
-For the cases when prefixing is not enough, contains the whole name of the sub-feature. ( A
-sub-feature may have a completely different name in some older version.
+#### `alternative_name`
+In some cases features are named entirely different and not just prefixed. Example:
```
Comment (wbamberg) [inline compat-data-schema.md]: different->differently

File: compat-data-schema.md [hunk 8]
```diff
@@ -1,334 +1,306 @@
...
+additional information. If there is only one entry in the array,
+the array can be a just a string. Example:
+
+* Indicating a restriction:
+```json
+{
+  "version_added": "3.5",
+  "notes": [
+    "Does not work on ::first-letter pseudo-elements.",
+    "See bug 1234567"
```
Comment (wbamberg) [inline compat-data-schema.md]: Note should end with a period. Also, maybe make this a link?

File: README.md [hunk 9]
```diff
@@ -1,55 +1,64 @@
...
+```
+npm install mdn-browser-compat-data
+```
+
+## Usage
+```js
+const bcd = require('mdn-browser-compat-data');
+bcd.data.css.properties.background;
+// returns a compat data object (see schema)
+```
```
Comment (Elchi3) [inline README.md]: This readme is also the readme appearing here: https://www.npmjs.com/package/mdn-browser-compat-data. I saw this quite often on other GitHub repositories/npm packages, too.

File: README.md [hunk 10]
```diff
@@ -1,55 +1,64 @@
-# browser-compat-data
+# mdn-browser-compat-data
```
Comment (Elchi3) [inline README.md]: I did this because our package is named "mdn-browser-compat-data".

File: compat-data-schema.md (hunk 5 above)
Comment (Elchi3) [inline compat-data-schema.md]: Might be a good follow-up to add more examples generally.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+```js
+ajv validate -s compat-data.schema.json -d "css/**/*.json"
+```
+We are planning to add proper test runner soon. Stay tuned!
+
+## Reviewer's checklist
+Not everything is enforced or validated by the schema. A few things a reviewer should pay attention to:
+
+* Feature identifiers (the data namespaces, like `css.properties.background`) should make sense and are spelled correctly.
+* Distinction between features and sub-features makes sense.
```
Comment (Elchi3) [inline CONTRIBUTING.md]: Now linked to the explanation in compat-data-schema.md.

**Comment by Elchi3** [comment]
Thanks so much @wbamberg! Really great review! This makes we want to have your reviews for other docs I'm writing, too.
